arcgis-ios-sdk-samples/Utility network/Display content of utility network container/DisplayContentUtilityNetworkContainerViewController.swift

```diff
@@ -0,0 +1,360 @@
+            guard let feature = sublayerResult.geoElements.first(where: { $0 is AGSArcGISFeature }) as? AGSArcGISFeature else { return }
+            addElementAssociations(for: feature)
+        }
+    }
+    
+    /// Get the containment associations from the chosen element.
+    ///
+    /// - Parameter containerFeature: The selected container feature
+    func addElementAssociations(for containerFeature: AGSArcGISFeature) {
+        // Create a container element using the selected feature.
+        guard let containerElement = utilityNetwork?.createElement(with: containerFeature) else { return }
+        // Get the containment associations from this element to display its content.
+        utilityNetwork?.associations(with: containerElement, type: .containment) { [weak self] containmentAssociations, _ in
+            // Determine the type of each element and add it to the array of content elements.
+            guard let self = self, let associations = containmentAssociations else { return }
+            let contentElements: [AGSUtilityElement] = associations.map { association in
+                if association.fromElement.objectID == containerElement.objectID {
+                    return association.toElement
+                } else {
+                    return association.fromElement
+                }
+            }
+            // If there are elements, find their corresponding features.
+            if !contentElements.isEmpty {
+                self.addGraphicsToFeatures(within: containerElement, for: contentElements)
+            }
+        }
+    }
+    
+    /// Get the features that correspond to the content's elements and add graphics.
+    ///
+    /// - Parameters:
+    ///     - containerElement: The container element of the selected feature
+    ///     - contentElements: An array of the content elements.
+    func addGraphicsToFeatures(within containerElement: AGSUtilityElement, for contentElements: [AGSUtilityElement]) {
+        // Save the previous viewpoint.
+        previousViewpoint = mapView.currentViewpoint(with: .boundingGeometry)
+        // Hide the layers to display the container view.
+        (mapView.map?.operationalLayers as? [AGSLayer])?.forEach { layer in
+            layer.isVisible = false
+        }
+        // Get the corresponding features for the array of content elements.
+        utilityNetwork?.features(for: contentElements) { [weak self] (contentFeatures, error) in
+            guard let self = self else { return }
+            if let contentFeatures = contentFeatures {
+                // Create and add a symbol for each of the features.
+                contentFeatures.forEach { content in
+                    let symbol = (content.featureTable as? AGSArcGISFeatureTable)?.layerInfo?.drawingInfo?.renderer?.symbol(for: content)
+                    self.graphicsOverlay.graphics.add(AGSGraphic(geometry: content.geometry, symbol: symbol))
+                }
```

Comment:
Better to create the graphics and add them all at once than to add them one at a time.
```suggestion
                let graphics: [AGSGraphic] = contentFeatures.compactMap { (feature) in
                    guard let featureTable = feature.featureTable as? AGSArcGISFeatureTable,
                          let symbol = featureTable.layerInfo?.drawingInfo?.renderer?.symbol(for: feature) else {
                        return nil
                    }
                    return AGSGraphic(geometry: feature.geometry, symbol: symbol)
                }
                self.graphicsOverlay.graphics.addObjects(from: graphics)
```